### PR TITLE
add region support (optional)

### PR DESCRIPTION
**Pull request by ottob**

### CODE REVIEW

File: geocode.go [hunk 1]
```diff
@@ -14,13 +14,17 @@ import (
...
 	client *http.Client
 	apiKey string
+	region string
 }
 
 // NewGoogleGeo returns a new GoogleGeo instance
-func NewGoogleGeo(apiKey string) *GoogleGeo {
+// The region parameter is optional, read about it here:
+// https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes
+func NewGoogleGeo(apiKey, region string) *GoogleGeo {
```
Comment (martinlindhe) [inline geocode.go]: Please leave existing NewGoogleGeo as is, but instead add a separate function such as "NewGoogleGeoWithRegion". This way we keep backward compatibility and the tests and README would still be correct.

Comment (ottob) [reply geocode.go]: Sure, either that or `func (g *GoogleGeo) SetRegion(region string)`. What do you prefer?

Comment (martinlindhe) [reply geocode.go]: Right! SetRegion() would be even better :-)

**Comment by martinlindhe** [comment]
Looks great, thanks!

**Comment by ottob** [comment]
Having thought about this some more it should probably be something like this instead:

`func (g *GoogleGeo) GeocodeWithRegion(address, region string) (*Point, error) {`

Since I want to reuse a single geocode client for multiple go routines but do lookups with different regions using this client.

Can you revert this? Or should I do that in a followup PR?

**Comment by martinlindhe** [comment]
Feel free to submit another PR and we'll sort it! No need to revert this change

**Pull request merged**